roles/confluent.common/tasks/main.yml

```diff
@@ -34,17 +34,17 @@
     owner: "{{ omit if archive_owner == '' else archive_owner }}"
     state: directory
     mode: 0755
-  when: installation_method == "archive"
+  when: installation_method == "archive" and privileged_install|bool
```

Comment:
So the idea is that the admin has to create this dir ahead of time? Isn't /opt/ available for random users?
I'm wondering if this should be another "omit the mode" case like the task below